<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeEnvDataDialogUI</class>
 <widget class="QWidget" name="ChangeEnvDataDialogUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>563</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Change Env Data</string>
  </property>
  <widget class="QTextEdit" name="commentEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>401</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="useModCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>87</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>use mods</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>520</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>520</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="ckomment_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Comment</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="name_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_major">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>57</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_minor">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>57</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_patch">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>57</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="version_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Version</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
